src/zenml/models/v2/core/pipeline_namespace.py

```diff
@@ -0,0 +1,113 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Models representing pipeline namespaces."""
```

Comment:
This is a brand new feature, right? Why do we handle it here and not as a separate FR?

Reply:
No this was implemented before, probably just the file moved without git detecting it.